src/Sensors/Sentinel1/FileIO/Metadata/Sentinel1Metadata.jl

```diff
@@ -5,7 +5,17 @@ include("MetadataUtil.jl")
 ##################### Constructors  ##################
 ######## Constructors for all the structures ########
 ######################################################
+function get_reference_time(meta_dict)::DateTime
+    start_time = meta_dict["product"]["adsHeader"]["startTime"]
+    reference_time = DateTime(start_time[1:19])
```

Comment:
always 1:19?

Reply:
Yes. It is the date format used in the metadata file